src/components/editMeasure/testCases/components/editTestCase/calculator/EditorCalculator.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Calculate from "./calculator-icon.svg";
+import { IconButton } from "@mui/material";
+import Tooltip from "@mui/material/Tooltip";
+
+const EditorCalculator = () => {
+  const toggleCalculator = () => {
+    const event = new CustomEvent("toggleEditorCalculatorBox");
+    window.dispatchEvent(event);
```

Comment:
I am not suggesting this change, but what was the motivation for using dispatchEvent vs a callback function?